src/RoleObjects/MenuItemData.js

```diff
@@ -148,7 +148,7 @@ export default class MenuItemData extends AccessibilityObject {
   /**
    * Keydown listener for when the item manages opening a menu
    * @access private
-   * @param {SyntheticEvent} evt - React event
+   * @param {SyntheticEvent} evt - event
```

Comment:
- these are no longer type `SyntheticEvent` - that's React
- just `Event` should do it